test/srt/test_bench_serving.py

```diff
@@ -97,8 +97,8 @@ def test_online_latency_default(self):
 
         if is_in_ci():
             self.assertLess(res["median_e2e_latency_ms"], 12000)
-            self.assertLess(res["median_ttft_ms"], 80)
-            self.assertLess(res["median_itl_ms"], 11)
+            self.assertLess(res["median_ttft_ms"], 86)
```

Comment:
QQ Why Median TTFT becomes higher?

Reply:
Because we delayed the processing of the first batch in favor of launching the kernels for the next batch first.